cluster-provision/centos8/scripts/dnsmasq.sh

```diff
@@ -18,7 +24,7 @@ for i in $(seq 1 ${NUM_NODES}); do
   for s in $(seq 1 ${NUM_SECONDARY_NICS}); do
     tap_name=stap$(($i - 1))-$(($s - 1))
     ip tuntap add dev $tap_name mode tap user $(whoami)
-    ip link set $tap_name master br0
+    ip link set $tap_name master br${s}
```

Comment:
Is it ok that those bridges are not connected outside?

Reply:
you can connect from eth1 to eth1 at different nodes (they use the same bridge)
What you cannot do is communicate from eth1 to eth2 and I think that we don't care for that, also we loose dhcp (I think) at knmstate is find and at kubevirt we don't use secondary nics at all, is kubemacpool in need for them ?

Reply:
I dont familiar with secondary nics usage for cluster nodes in KMP, 
@RamLavi do you have any idea if we use secondary nics in kmp?

Reply:
well yeah, kmp allows to allocate mac addresses from a pool to secondary interfaces using Network Plumbing Working Group de-facto standard.